charts/sonarqube-dce/templates/prometheus-podmonitor.yaml

```diff
@@ -34,4 +38,28 @@ spec:
       name: {{ template "sonarqube.fullname" . }}-monitoring-passcode
       key: SONAR_WEB_SYSTEMPASSCODE
     {{- end }}
+{{- if .Values.ApplicationNodes.prometheusExporter.enabled}}
+{{- if .Values.ApplicationNodes.prometheusExporter.ceBeanPort}}
+  - port: monitoring-ce
+    path: /
+    scheme: http
+    {{- if .Values.ApplicationNodes.prometheusMonitoring.podMonitor.interval }}
+    interval: {{ .Values.ApplicationNodes.prometheusMonitoring.podMonitor.interval }}
+    {{- end }}
+    {{- if .Values.ApplicationNodes.prometheusMonitoring.podMonitor.scrapeTimeout }}
+    scrapeTimeout: {{ .Values.ApplicationNodes.prometheusMonitoring.podMonitor.scrapeTimeout }}
```

Comment:
Should those endpoints also use `SONAR_WEB_SYSTEMPASSCODE` feature ? or do they require no authentication ?

Reply:
Nope, they don't require authentication... The only endpoint that requires authentication is the Prometheus native one...